src/CarbonAware.Tools/CarbonAware.Tools.WattTimeClient/src/WattTimeServerMocks.cs

```diff
@@ -0,0 +1,145 @@
+using System.Net;
```

Comment:
Should this be in with the tests rather than in with the production code since its a mock?

Reply:
Agree with @rakker91 comment. These tests should be like an independent project, so having files touched or things under the current projects of the app, immediately is raises a flag.

Reply:
My reason for putting it in the WattTime folder is because it needs to access the WattTime paths which are currently fully internal to WattTime. Happy to change the access level to that and import that, and the whole watttime project into the integration test project if y'all think that's best

Reply:
You might consider making the WattTime paths internal and then adding an InternalsVisibleTo attribute to give access to those paths to the mock.

Reply:
Cool, will move. Thanks!